base/libgit2/tree.jl

```diff
@@ -122,3 +131,47 @@ function Base.show(io::IO, tree::GitTree)
     println(io, "Owner: ", repository(tree))
     println(io, "Number of entries: ", count(tree))
 end
+
+"""
+    getindex(tree::GitTree, target::AbstractString)::GitHash
```

Comment:
Base still uses `->` to show return types in docstrings

Reply:
why?

Reply:
Partly historical artifact, partly because it's often used in such a way that the `:: T` return type syntax would be wrong or inappropriate.